runtime/io.c

```diff
@@ -532,12 +547,14 @@ CAMLprim value caml_ml_out_channels_list (value unit)
   for (channel = caml_all_opened_channels;
        channel != NULL;
        channel = channel->next)
-    /* Testing channel->fd >= 0 looks unnecessary, as
+    /* Include only output channels opened from OCaml and not closed yet.
+       Testing channel->fd >= 0 looks unnecessary, as
        caml_ml_close_channel changes max when setting fd to -1. */
-    if (channel->max == NULL) {
+    if (channel -> max == NULL
```

Comment:
```suggestion
    if (channel->max == NULL
```
?

Reply:
Oups!  Apologies for this strange typo.  I guess those spaces were destined for another window but windows focus wasn't where I expected it...  I squashed the fix with the original commit, so that it doesn't show up.